website/docs/r/synthetics_private_location.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_private_location"
+sidebar_current: "docs-newrelic-resource-synthetics-private-location"
+description: |-
+Create and manage Synthetics private location in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_private\_location
+
+Use this resource to create and manage New Relic Synthetic private location.
+
+## Example Usage
+
+```hcl
+resource "newrelic_synthetics_private_location" "bar" {
+ accountId = "NewRelic account ID"
+ description = "The private location description"
+ name = "The name of the private location"
+ verifiedScriptExecution = "true or false"
+}
+```
+
+## Argument Reference
```

Comment:
You will need to update this after you processed some of the other comments in code. The description is not optional for example.

Reply:
Changed the Description